lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -134,3 +164,14 @@ DetailsRenderer.ListDetailsJSON; // eslint-disable-line no-unused-expressions
  * }}
  */
 DetailsRenderer.CardsDetailsJSON; // eslint-disable-line no-unused-expressions
+
+/**
+ * @typedef {{
+ *     type: string,
+ *     text: string,
```

Comment:
what is `text` in this case? like a description of the code snippet that follows?